crates/ruff_python_formatter/src/expression/expr_compare.rs

```diff
@@ -7,7 +7,7 @@ use crate::comments::Comments;
 use ruff_formatter::{write, Buffer, FormatResult};
 use rustpython_parser::ast::ExprCompare;
 
-#[derive(Default)]
+#[derive(Default, Debug)]
```

Comment:
is this still needed?

Reply:
i guess not though it was handy. any reason not to keep it? (can make a separate commit if that's the issue)

Reply:
it generates unnecessary code; I'm fine with adding the std derives on everything but we have kept only the necessary ones in the rest of the codebase and it should also be better for compile time